draft-packages/form/KaizenDraft/Form/Primitives/Label/Label.module.scss

```diff
@@ -3,7 +3,7 @@
 @import "~@kaizen/design-tokens/sass/color";
 @import "../../variables";
 
-$label-start-margin: 10px;
+$label-start-margin: $spacing-xs;
```

Comment:
One thing we did notice was that this affects not only Checkbox, but also RadioField and ToggleSwitchField. Also if any consumers are using this component directly in their repos.